ritcovid.py

```diff
@@ -96,47 +96,37 @@ def get_statistics():
     return all_students, all_staff, new_students, new_staff, campus_quarantine, offcampus_quarantine, campus_isolated, \
            offcampus_isolated, isolation_beds, last_updated, new_case_stat, tests_administered
 
-def check_last_known():
-    # to anyone looking at this, this is probably not the best way to do this
-    # but it's temporary so shhh.
 
+def check_last_known():
+    # Check to see if the file does not exist or is empty.
     if os.path.exists("last_known.txt") == False or os.path.getsize("last_known.txt") == 0:
```

Comment:
This conditional is not needed. See comment on line 106.

Reply:
The purpose of this conditional is to check if the file exists and isn't empty when initially loading the bot. We don't want it overriding the what is there if the file already exists.